test/claims-status/components/appeals-v2/Issues.unit.spec.jsx

```diff
@@ -21,8 +22,13 @@ describe('<Issues/>', () => {
     expect(wrapper.find('CollapsiblePanel').length).to.equal(0);
   });
 
-  it('should render one panel when only an open *OR* a closed issue is passed in', () => {
-    const wrapper = shallow(<Issues {...oneIssues}/>);
+  it('should render one panel when only an open issue is passed in', () => {
+    const wrapper = shallow(<Issues {...oneOpenIssue}/>);
+    expect(wrapper.find('CollapsiblePanel').length).to.equal(1);
+  });
```

Comment:
I hate to have you keep iterating on this, but can we make sure it's the right panel? Something like
```js
const panels = wrapper.find('CollapsiblePanel');
expect(panels.length).to.equal(1);
expect(panels.first().text()).to.contain('Currently on appeal');
```